doc/Dataplane Code Gen/DataplaneCodeGen_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,185 @@
+
+**Learn more**: see the [OpenAPI Swagger Documentation](https://swagger.io/docs/) to learn more about swagger file formats.
+
+#### 4. Save and commit your changes to your branch
+
+This will make it easier to see the changes you make when you re-generate the code later.
+
+Congratulations, you've generated your first Azure SDK library! Next, we'll discuss how we can add tests, samples and documents.
+
+## Tests
+
+In this section, we will talk about adding unit tests and live tests and how to run them.
+
+- Add back `Azure.Template.DataplaneCodeGen.Tests` csproj in your `Azure.<your-sdk-name>.sln`.
+- Rename the `Azure.Template.DataplaneCodeGen.Tests` project to `Azure.<your-sdk-name>.Tests`.
+- Rename the `TemplateServiceTestEnvironment.cs` file to `<client-name>TestEnvironment.cs` and update the test environment. Before running live tests you will need to create live test resources, please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/README.md) to learn more about how to manager resources and update test environment.
+- Rename the `TemplateServiceLiveTests.cs` file to `<client-name>LiveTests.cs` and remove all the template project tests. Please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#live-testing) to add live tests.
+- Rename the `TemplateServiceTests.cs` file to `<client-name>Tests.cs` and remove all the template project tests and add all the unit tests in this file.
+- Remove the `SerializationHelpers.cs` file.
+
+**Note**: `Azure.<your-sdk-name>` is the Azure SDK package name and `<client-name>` is a client name, C# generator will generate a client which you can find in `Azure.<your-sdk-name>/Generated` directory.
+
+**Learn more:** see the [docs](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#to-test-1) to learn more about tests.
+
+## Samples
+
+In this section, we will talk about how to add samples. As you can see, we already have a `Samples` folder under `Azure.<your-sdk-name>/tests` directory. We run the samples as a part of tests. First, rename the `TemplateServiceSamples.HelloWorld.cs` file to `<client-name>Samples.HelloWorld.cs` and remove the existing sample tests. You will add the basic sample tests for your SDK in this file. Create more test files and add tests as per your scenarios.
+
+**Learn more:** For general information about samples, see the [Samples Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples).
+
+## Snippets
+
+Snippets are the great way to reuse the sample code. Please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) to add snippets in your samples.
```

Comment:
Would it be possible to call out that snippets allow us to verify that the code in our samples and READMEs is always up to date, and passes unit tests?